resolwe/flow/expression_engines/jinja/filters.py

```diff
@@ -0,0 +1,56 @@
+"""Filters for Jinja expression engine."""
+import os
+
+from resolwe.flow.models import Data
+
+
+def _get_data_attr(data, attr):
+    """Get data object field."""
+    if isinstance(data, dict):
+        # `Data` object's id is hydrated as `__id` in expression engine
+        data = data['__id']
+
+    data_obj = Data.objects.get(id=data)
+
+    return getattr(data_obj, attr)
+
+
+def name(data):
+    """Return `name` of `Data`."""
+    return _get_data_attr(data, 'name')
+
+
+def id(obj):  # pylint: disable=redefined-builtin,invalid-name
+    """Return ``id`` key of dict."""
+    return obj['__id']
+
+
+def type(obj):  # pylint: disable=redefined-builtin
```

Comment:
I've just copied this over from previous `templatetags` and did not change the implementation at all. So, should we change it?